weltyt.py

```diff
@@ -44,6 +44,33 @@ def buy_plants(plants):
 
         print("Invalid input, write integer only please. ")
 
+def buy_pesticides(pesticides):
+    amount = input("How many pesticides do you want to buy? Write only integer please. ")
+
+    try:
+        amount = int(amount)
```

Comment:
We want to catch only this line, not the whole logic after it.

Reply:
But we have to. Otherwise it doesn't work. We tried it before.